geometry_msgs/PoseStamped pose
visualization_msgs/Marker[] markers
---
geometry_msgs/PoseStamped pose
